encoding/rs/params.go

```diff
@@ -37,7 +37,7 @@ func (p EncodingParams) Validate() error {
 }
 
 func GetNumSys(dataSize uint64, chunkLen uint64) uint64 {
-	dataLen := RoundUpDivision(dataSize, bls.BYTES_PER_COEFFICIENT)
+	dataLen := RoundUpDivision(dataSize, encoding.BYTES_PER_COEFFICIENT)
```

Comment:
Noting a duplicate roundupdivision to remove in a separate PR.

Reply:
what do you mean, there is only one RoundUpDivision, except the one in encoding/test/main.go. I will remove the one in main.go

Reply:
I see, there is one using generics, good

Reply:
Let's do it in a separate PR